src/krasa/grepconsole/model/GrepColor.java

```diff
@@ -1,11 +1,15 @@
 package krasa.grepconsole.model;
 
+import com.intellij.openapi.editor.colors.ColorKey;
+import com.intellij.openapi.editor.colors.EditorColorsUtil;
+
 import java.awt.*;
 
 public class GrepColor extends DomainObject {
 
 	private boolean enabled;
 	private Integer color;
+  private String colorKey;
```

Comment:
This felt like the easiest path forward because the existing users' settings will remain the same. This setting comes into play for new users or users who restore to the default settings.